submissions/insertion_sort/InsertionSort.py

```diff
@@ -0,0 +1,14 @@
+def InsertionSort(arr):
+    for i in range(1, len(arr)):
```

Comment:
Generally try to avoid single letter variables because it makes the code a little hard to read. A short descriptive name might be a little better here.